daffodil-core/src/test/java/org/apache/daffodil/jexample/TestAPI.java

```diff
@@ -918,11 +919,11 @@ public void testAPI20() throws Exception {
       parseXMLReader.setContentHandler(contentHandler);
       parseXMLReader.setErrorHandler(errorHandler);
       parseXMLReader.setProperty(DaffodilParseXMLReader.DAFFODIL_SAX_URN_BLOBDIRECTORY,
-        Paths.get(System.getProperty("java.io.tmpdir")));
+          Paths.get(System.getProperty("java.io.tmpdir")));
```

Comment:
I think we still use 2-space indentation for .java files? Should we undo these indentation changes?